spotguide/spotguide.go

```diff
@@ -48,7 +48,6 @@ const PipelineYAMLPath = ".banzaicloud/pipeline.yaml"
 const ReadmePath = ".banzaicloud/README.md"
 const IconPath = ".banzaicloud/icon.svg"
 const CreateClusterStep = "create_cluster"
-const DeployApplicationStep = "deploy_application"
```

Comment:
This change is totally unrelated here. Is it by accident?

Reply:
> The change itself is not related, could be in a different commit.

I think it should be a separate PR, otherwise we won't know why it changed. CS fixes and stuff like that doesn't matter, but this is totally unrelated here.

Reply:
We discussed it privately: the next time we have a confusion because of an unrelated change, @tothandras pays lunch for a week for everyone. 😄